board/app/models/follow.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+class Follow extends AppModel 
+{
+
+    public function addFollowing()
+    { 
+        try {
+            $db = DB::conn();
+            $db->begin();
+            $params = array(
+            'username' => $this->username,
+            'user_id' => $this->user_id
+            );
```

Comment:
Done and applied to others po. Thank you :bow:
--> https://github.com/maragatchalian/FoodFacts/commit/3d90ceb5bfe86bf2a5e3dfaee3b5cbce5df1ee37